nsq/async.py

```diff
@@ -277,7 +277,7 @@ def _read_body(self, data):
             self.trigger(event.DATA, conn=self, data=data)
         except Exception:
             logger.exception('uncaught exception in data event')
-        self.io_loop.add_callback(self._start_read)
```

Comment:
Yeah, that seems reasonable enough. I wonder if there's a more efficient way of doing this then?